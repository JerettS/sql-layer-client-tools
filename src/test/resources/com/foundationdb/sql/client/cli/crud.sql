CREATE TABLE t(id INT, v VARCHAR(32));
SELECT * FROM t;
INSERT INTO t VALUES (1, 'foo'), (2, 'bar'), (3, 'zap');
SELECT * FROM t;
UPDATE t SET v='pop' WHERE id=1;
DELETE FROM t WHERE id=3;
SELECT id col1, v col2 FROM t;
