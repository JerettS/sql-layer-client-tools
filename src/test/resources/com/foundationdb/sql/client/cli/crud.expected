ROWS: 0

 id | v 
----+---
(0 rows)

ROWS: 3

 id |  v  
----+-----
  1 | foo 
  2 | bar 
  3 | zap 
(3 rows)

ROWS: 1

ROWS: 1

 col1 | col2 
------+------
    1 | pop  
    2 | bar  
(2 rows)

